Cargo.toml

```
@@ -34,7 +34,7 @@ wws-server = { workspace = true }
 wws-project = { workspace = true }
 
 [dev-dependencies]
-reqwest = { version = "0.11", features = ["blocking"] }
+reqwest = { version = "0.11", features = ["rustls", "blocking"] }
```

```suggestion
reqwest = { version = "0.11", features = ["rustls-tls", "blocking"] }
```

As per https://github.com/seanmonstar/reqwest/blob/master/Cargo.toml#L41

---

We might want to use `rustls-tls-webpki-roots` as well, so that we can provide a static binary that contains the root certificates bundled in the binary itself, without the need for the host system running wws to have ca-certificates or similar installed.

This has its problems of course, as for example, root certificate revocation, but I think it should be fine for now.

---

Good catch! Unfortunately, I'm going to close this PR due to #256. We still need to find an alternative that works for git2-rs :/

---

Ouch, right.